testsuite/MDAnalysisTests/coordinates/test_chemfiles.py

```diff
@@ -23,22 +23,42 @@
 import pytest
 
 import MDAnalysis as mda
-from MDAnalysis.coordinates.chemfiles import (
-    ChemfilesReader, ChemfilesWriter, check_chemfiles_version,
-)
+from MDAnalysis.coordinates.chemfiles import ChemfilesReader, ChemfilesWriter
+from MDAnalysis.coordinates.chemfiles import check_chemfiles_version
 
 from MDAnalysisTests import datafiles
 from MDAnalysisTests.coordinates.base import (
-    MultiframeReaderTest, BaseWriterTest, BaseReference
+    MultiframeReaderTest,
+    BaseWriterTest,
+    BaseReference,
 )
 from MDAnalysisTests.coordinates.test_xyz import XYZReference
 
 
 # skip entire test module if no appropriate chemfiles
-chemfiles = pytest.importorskip('chemfiles')
-@pytest.mark.skipif(not check_chemfiles_version(),
-                    reason="Wrong version of chemfiles")
-class TestChemFileXYZ(MultiframeReaderTest):
+chemfiles = pytest.importorskip("chemfiles")
+
+
+class TestChemfileVersion(object):
+    def test_version_check(self):
+        # Make sure the version check works as intended
+        import chemfiles
+
+        actual_version = chemfiles.__version__
+        chemfiles.__version__ = "0.8.3"
+        assert not check_chemfiles_version()
+
```

Comment:
Ok I see what's going on here (and why coverage isn't improving), technically we could use `mock` but given that 2.0 is around the corner we can just go with a quick solution and implement a more elegant one later if necessary.

Something like this should cover the two missing sections:

```suggestion
        with pytest.raises(RuntimeError, match='Please install Chemfiles > 0.9'):
            ChemfilesReader('test.nc')

        with pytest.raises(RuntimeError, match='Please install Chemfiles > 0.9'):
            ChemfilesWriter('test.nc')

```

Reply:
oh, right my bad, I was testing the wrong code =(. I'll integrate this today!

Reply:
@Luthaf would you be able to make any changes by end of Friday? We're trying to get 2.0 out and will freeze develop on 2021-05-07.

Reply:
Done, I hope the coverage will be good!